fineract-core/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java

```diff
@@ -293,7 +298,12 @@ private void publishHookEvent(final String entityName, final String actionName,
                 } else if (result instanceof ErrorInfo ex) {
                     reqmap.put("status", "Exception");
 
+                    if (!isValidJson(ex.getMessage(), type)) {
```

Comment:
This is wrong... you should do the following:
wrap around the `gson.fromJson(ex.getMessage(), type)`
```
Map<String, Object> errorMap;
try {
       errorMap = gson.fromJson(ex.getMessage(), type);
} catch (Exception e) {
        errorMap.put("errorMessage", ex.getMessage());
  }
errorMap.put("errorCode", ex.getErrorCode());
errorMap.put("statusCode", ex.getStatusCode());
   ```

Reply:
Okay, I think this makes more sense.
But for testing, does the existing tests are enough or do we need to add some other tests? (given that `publishHookEvent()` is private)